### DISCUSSION TITLE
Externally Closed School Email and Verification

### DISCUSSION THREAD

**Discussion by mertkan-k** [community/community#64721] [GitHub Education]
### Select Topic Area

Product Feedback

### Body

Hello guys, i can't receive external emails on my school e-mail. How do i verification that i am a student?

**Comment by phucho0237** [comment]
Do you try to find it in Spam folder?

**Comment by cdxgh** [comment]
I'm sorry to hear that you're having trouble receiving verification emails for your school email. Not receiving verification emails can be frustrating, but there are several steps you can take to troubleshoot and resolve this issue:

1. **Check Spam Folder:** Sometimes, verification emails may end up in your spam or junk folder. Check these folders in your email client to see if the verification email was mistakenly categorized as spam.

2. **Whitelist the Sender:** Make sure that the sender's email address or domain is added to your email client's whitelist or safe senders list. This can prevent the email from being marked as spam.

3. **Wait and Retry:** Sometimes, there may be delays in email delivery. Wait for some time, and then request a new verification email if possible.

4. **Contact IT Support:** If you're using a school email address, your school's IT support or helpdesk team can be a valuable resource. They can check if there are any issues with email delivery or your email account.

5. **Check Email Quota:** Ensure that your email inbox is not full or over its storage limit. If your inbox is full, you may not be able to receive new emails.

6. **Correct Email Address:** Double-check that you've entered the correct email address when requesting the verification email. Even a small typo can prevent you from receiving the email.

7. **Blocked Senders:** Check if you have inadvertently blocked the sender's email address or domain in your email settings.

8. **Server Issues:** Sometimes, email servers can experience issues. If this is the case, it may be affecting email delivery. Your school's IT team should be able to investigate and resolve server-related issues.

9. **Verify Mail Forwarding:** If you have set up email forwarding or redirection, ensure that it is configured correctly and not causing email delivery problems.

10. **Alternate Email Address:** If possible, provide an alternate email address for verification. This can be a personal email address that you have more control over.

11. **Contact the Sending Service:** If the verification email is coming from a specific service (e.g., an online platform or application), consider reaching out to their support team for assistance. They may be able to resend the verification email or provide additional guidance.

12. **Email Client Settings:** Check your email client settings to ensure that they are configured correctly, especially if you are using email forwarding or filtering rules.

If none of these steps resolve the issue, I recommend reaching out to your school's IT support team as they have access to the email system and can provide more specific assistance tailored to your school's email setup. They can also check for any issues on the school's email server that may be causing the problem.

**Comment by mertkan-k** [comment]
Thanks for answers.
At the same time i have been tring to send test mail from my gmail and i doens't receive as well. I did some research and they(not school IT) say like it is close for external e-mails, i don't know how can i verify my student pack. Is it enough to select my personal e-mail and take photo of my ID for verification? And even if it it enough for student pack, will it enough for other apps like GitKraken?

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @mertkan-k , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!
